websites/V/Voxiom/dist/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.6",
+  "author": {
+    "name": "Meow",
+    "id": "521701910609133570"
+  },
+  "service": "Voxiom",
+  "description": {
+    "en": "English"
```

Comment:
English?

Reply:
thought that was a description of the language for some reason, ill fix right now